CMakeLists.txt

```diff
@@ -6,6 +6,9 @@ set (RFR_VERSION_MAJOR 0)
 set (RFR_VERSION_MINOR 8)
 set (RFR_VERSION_RELEASE 1)
 
+# Print debug information about boost
```

Comment:
Hey, is this still necessary after you got the whole installation process running?

Reply:
This will allow us to understand from where boost is taken, particularly this just prints:
```
-- Boost toolset is gcc9 (GNU 9.3.0)
-- Scanning /usr/lib/x86_64-linux-gnu/cmake/boost_unit_test_framework-1.71.0/libboost_unit_test_framework-variant*.cmake
--   Including /usr/lib/x86_64-linux-gnu/cmake/boost_unit_test_framework-1.71.0/libboost_unit_test_framework-variant-shared.cmake
--   [x] libboost_unit_test_framework.so.1.71.0
--   Including /usr/lib/x86_64-linux-gnu/cmake/boost_unit_test_framework-1.71.0/libboost_unit_test_framework-variant-static.cmake
--   [ ] libboost_unit_test_framework.a
-- Adding boost_unit_test_framework dependencies: headers
-- Found Boost: /usr/lib/x86_64-linux-gnu/cmake/Boost-1.71.0/BoostConfig.cmake (found version "1.71.0") found components: unit_test_framework 
```
This was important to debug why boost was not red, even though virtual machines come with this. I can remove it if you think this print statement is not needed, and if this ever fail again we can just add it back as needed.

Reply:
If you think this statement is vital we can just leave it in